README.md

```diff
@@ -56,88 +56,147 @@ Travis CI testing.
+## Deploy With Helm
 
-* Deploy [ApiGateway](kubernetes/apigateway)
-* Deploy [Zookeeper](kubernetes/zookeeper)
-* Deploy [Kafka](kubernetes/kafka)
-* Deploy [Controller](kubernetes/controller)
-* Deploy [Invoker](kubernetes/invoker)
-* Deploy [Nginx](kubernetes/nginx)
+Deployment can be done by using the following single command:
+```shell
+helm install . --namespace=openwhisk -f mycluster.yaml
+```
 
-## Install system actions and the openwhisk catalog
+After a while, if you can see all the pods listed by the command below
+are in `Running` state, congratulations, you have finished OpenWhisk
+deployment:
+```shell
+kubectl get pods -n openwhisk
+```
+
+You can also use `helm status` to get a summary of the various
+Kubernetes artifacts that make up your OpenWhisk deployment.
 
-* Install [RouteMgmt](kubernetes/routemgmt)
-* Install [Package Catalog](kubernetes/openwhisk-catalog)
+## Configure the wsk CLI
 
-## Verify
+Configure the OpenWhisk CLI, wsk, by setting the auth and apihost
+properties (if you don't already have the wsk cli, follow the
+instructions [here](https://github.com/apache/incubator-openwhisk-cli)
+to get it). Replace `whisk.ingress.api_host_name` and `whisk.ingress.api_host_port`
+with the actual values from your mycluster.yaml.
+```shell
+wsk property set --apihost whisk.ingress.api_host_name:whisk.ingress.api_host_port
+wsk property set --auth 23bc46b1-71f6-4ed5-8c54-816aa4f8c502:123zO3xZCLrMN6v2BKK1dXYFpXlPkccOFqm12CdAsMgRU4VrNZ9lyGVCGuMDGIwP
+```
+
+## Verify your OpenWhisk Deployment
 
 Your OpenWhisk installation should now be usable.  You can test it by following
 [these instructions](https://github.com/apache/incubator-openwhisk/blob/master/docs/actions.md)
 to define and invoke a sample OpenWhisk action in your favorite programming language.
 
-Note: if you installed self-signed certificates when you configured Nginx, you will need to use `wsk -i` to suppress certificate checking.  This works around `cannot validate certificate` errors from the `wsk` CLI.
+Note: if you installed self-signed certificates, which is the default
+for the OpenWhisk Helm chart, you will need to use `wsk -i` to
+suppress certificate checking.  This works around `cannot validate
+certificate` errors from the `wsk` CLI.
 
-# Cleanup
+If your deployment is not working, check our
+[troubleshooting guide](./docs/troubleshooting.md) for ideas.
 
-At some point there might be a need to cleanup the Kubernetes environment.
-For this, we want to delete all the OpenWhisk deployments, services, jobs
-and whatever else might be there. This is easily accomplished by
-deleting the `openwhisk` namespace and all persistent volumes labeled with
-pv-owner=openwhisk:
+# Cleanup
 
+Use the following command to remove the deployment:
+```shell
+helm delete <release_name>
 ```
-kubectl delete namespace openwhisk
-kubectl delete persistentvolume -lpv-owner=openwhisk
+or with a `--purge` option if you want to completely remove the deployment from helm:
```

Comment:
maybe explain wha't the difference between the commands `remove the deployment` vs. `completely remove the deployment`

Reply:
good point. done.